culture/review_culture.md

```diff
@@ -0,0 +1,142 @@
+
+### Review
+
+Once a change is ready and has been self-reviewed, it's ready for review. Since
+most SourceCred artifacts are tracked via repositories on GitHub, this will
+usually mean creating a pull request.
+
+Once the change is proposed, anyone in the project can add their own review.
+The person proposing the change can also request specific reviewers, if certain
+individiuals have context or knowledge thet will make their reivew particularly
+valuable.
+
+On finishing their review, the reviewer can:
+- ask questions to better understand the change
+- suggest changes to it
+- approve the change
+
+Most changes go through a few rounds of reviews before being approved.
+
+### Approval and Merging
+
+During the review process, a successful change will accumulate "approvals" from
+satisfied reviewers. At some point, the change will have "enough approvals" and
+will be ready to merge. There's no hard-and-fast rule to determine when a
+change has enough approvals. Generally, simple and small changes are easy to
+approve, whereas changes that are breaking new ground, or have wide-ranging
+consequences, will require more reviews. If you're unsure as to whether a
+change is ready to merge, ask one of the maintainers.
+
+### Changes that never merge
+
+Some changes don't ever merge. A few good reasons why this may happen include:
+- Reviewers may find that the change isn't actually needed, is basically
+  unsound, or isn't aligned with the project's values or goals
+- Over the course of review, the contributor or reviewers may find a different
+  and better way to achieve the same goals
+
+# Best Practices for Reviewers
+
+## Be respectful and appreciative
+
+As a reviewer, you should be respectful and appreciative towards the person
+proposing the change. This is true especially when you quite disagree with the
+change itself. It's fine to express strong beliefs that a change is misguided,
+but you shouldn't make the contributor feel belittled.
+
+## Limit nit-picking
+
+Nit-picking is when a review focuses more on small, superficial details (e.g.
+precise formatting or word choice) rather than on the substance of the change.
+Nit-picking is a lot easier than substantive review, which can make it
+tempting. It's ok to nit pick in moderation; when nit-picking, it's polite to
+preface your suggestion with "nit", as in: "nit: this comma should actually be
+a semicolon". Avoid holding up a review over nits.
+
+## Avoid "not how I would have done it"
+
+As a reviewer, you'll often notice that someone has made a change differently
+from how you would have made it. It can be tempting to think that your way is
+_the_ right way, and to ask the contributor to change their approach. Before
+doing this, you should ask: in what ways is doing it my way materially better?
+What benefits come from the changed approach? If you don't have clear answers
+to these questions, don't ask for the change.
```

Comment:
_Adding this as a thought, not a request for change._

Personally I still regularly fall for this trap. But at the same time very much appreciate receiving a thoughtful "here's how you could have done it differently". There's so much I've learned from those. So I wouldn't want to discourage that sort of feedback.

If I had to capture my rule of thumb for this (sometimes difficult) distinction in a one-liner it would be:
Try to share your experience, without blocking or frustrating something that is an improvement.

If you think it's not an improvement, request a change.
If you think it has merit, but is not how you would have done it, caution on the side of approving it. Then follow up with your own additions afterwards if you really want to.

Reply:
Yeah, I think the distinction is between asking "did you consider doing it [my preferred way]", which is generally helpful, vs blocking the change saying "please re-do this my way", which is expensive. happy to clarify this in the guide...

Reply:
This is a common trap, and (in my experience) the most common source of
unpleasant experiences on the side of the proposer. The distinction
between a non-blocking ask and a blocking request is important, but
usually these kinds of comments just don’t need to be made at all. The
questions in the current draft (how is my suggestion _materially
better_, not just marginally cleaner?) strike me as the key ones.

I didn’t really realize how much of a difference this made until I
started regularly receiving reviews from one specific person, and
quickly learned to expect that every single comment on a review from
them would be actionable and provide a significant benefit. It totally
changes what it feels like to open a review, and I’ve since deliberately
adjusted the style of my own reviews to emulate this.

Reply:
Thanks for the feedback WC. Going to leave the prose as is for now; as ever, open to potential improvements down the line.